spec/policy/find_service/find_service_spec.lua

```diff
@@ -58,11 +61,39 @@ describe('find_service', function()
           assert.is_nil(context.service)
         end)
       end)
+
+      describe('and fallback to find by host is disabled', function()
+        before_each(function()
+          ConfigurationStore.path_routing = true
+          ConfigurationStore.path_routing_only = true
+        end)
+
+        it('stores nil in the context even if there is a service for the host', function()
+          local service = { id = '1' }
```

Comment:
A quick comment, shouldn't need to test with more than one service?

Reply:
It's not really needed. This module only cares about what the `HostBasedFinder` and the `PathBasedFinder` modules return and they just return one service. In those 2 modules we have tests with configurations that have multiple services, so I think we are covered.